roles/openshift_logging/tasks/label_node.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
By tools do you mean other ansible tasks or actually some external tool?  Please point us elsewhere if we should be using something else, otherwise logging will not actually collect logs unless the nodes are properly labeled